### PR TITLE
Adjust centaur to sub workflow changes and add sub workflow tests

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

**Comment by ruchim** [comment]
👍 Looks awesome, and I like the idea of using the <<IMPORT>> identifier in the WDLs for testing sub-workflows.

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/114/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/refresh_token_sub_workflow/refresh_token_sub_workflow.inputs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "refresh_token_sub_workflow.hello.iFile": "gs://centaur-refresh-private/inputs/taylorSwift.txt",
+  "refresh_token_sub_workflow.wf_hello.wfIFile": "gs://centaur-refresh-private/inputs/taylorSwift.txt"
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/refresh_token_sub_workflow/refresh_token_sub_workflow.inputs:3]: what the

File: src/main/resources/standardTestCases/sub_workflow_hello_world.test [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  wdl: sub_workflow_hello_world/sub_workflow_hello_world.wdl
+  imports: [
+    sub_workflow_hello_world/sub_workflow_hello_world_import.wdl
+  ]
+}
+
+metadata {
+  workflowName: main_workflow
+  status: Succeeded
+  "outputs.main_output": "Hello sub world!"
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/sub_workflow_hello_world.test:15]: 😄

**Review by mcovarr — APPROVED** [review]

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/114/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/refresh_token_sub_workflow/refresh_token_sub_workflow.inputs (hunk 1 above)
Comment (Horneth) [reply src/main/resources/standardTestCases/refresh_token_sub_workflow/refresh_token_sub_workflow.inputs:3]: I'ma afraid @ruchim is the one responsible here :p

**Comment by Horneth** [comment]
I'm going to wait for the Cromwell PR to be merged to merge this one or everyone's PRs will be red

File: src/main/resources/standardTestCases/sub_workflow_hello_world.test (hunk 2 above)
Comment (cjllanwarne) [reply src/main/resources/standardTestCases/sub_workflow_hello_world.test:15]: `Hello Hades`?

File: src/main/resources/standardTestCases/sub_workflow_hello_world.test [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: sub_workflow_hello_world
+testFormat: workflowsuccess
+
+files {
+  wdl: sub_workflow_hello_world/sub_workflow_hello_world.wdl
+  imports: [
+    sub_workflow_hello_world/sub_workflow_hello_world_import.wdl
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/sub_workflow_hello_world.test:8]: Is this working...?!

Comment (Horneth) [reply src/main/resources/standardTestCases/sub_workflow_hello_world.test:8]: Last time I tried yes it was - only for one layer of imports though (if your imported file contains an import too it won't work)

**Pull request merged**